pkg/kubelet/dockertools/manager.go

```diff
@@ -1940,44 +2083,130 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, _ api.PodStatus, podStatus *kubec
 				continue
 			}
 		}
+
 		glog.V(4).Infof("Creating container %+v in pod %v", container, format.Pod(pod))
-		err, msg := dm.imagePuller.PullImage(pod, container, pullSecrets)
-		if err != nil {
+		if err, msg := dm.tryContainerStart(container, pod, podStatus, pullSecrets, namespaceMode, pidMode, podIP); err != nil {
 			startContainerResult.Fail(err, msg)
+			utilruntime.HandleError(fmt.Errorf("container start failed: %v: %s", err, msg))
 			continue
 		}
+		//
```

Comment:
Weird, I thought all of these were fixed and I pushed the changes.

On Wed, May 4, 2016 at 10:22 PM, Prashanth B <notifications@github.com>
wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/kubernetes/kubernetes/pull/23567#discussion_r62142612>
> :
>
> >  			continue
> >  		}
> > +		//
>
> comment?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23567/files/3b557a832be1105abaebb33aeee07ab1ea9a8f43#r62142612>
>